common/rbnf/de_CH.xml

```diff
@@ -209,221 +209,10 @@ x.x: =#,##0.#=;
 %spellout-ordinal-m:
 -x: minus >>;
 x.x: =#,##0.#=;
-0: =%spellout-ordinal=m;
-]]></rbnfRules>
-            <!-- The following redundant ruleset elements have been deprecated and will be removed in the next release. Please use the rbnfRules contents instead. -->
```

Comment:
We are leaving in the old rules for one release, so they shouldn't disappear. Also, very odd to keep just one ("30").

Reply:
It's because of MINIMIZE rules. If transformed = original, it drops the value.

'30' has `dreissig` which is transformed from `dreißig` - that's the only modified item, the rest get dropped.